app/models.py

```diff
@@ -1039,6 +1039,10 @@ def validate(self, enforce_required=True, required_fields=None):
         ):
             errs['niceToHaveRequirements'] = 'answer_required'
 
+        if max_day_rate and 'dayRate' not in errs:
+            if float(self.data['dayRate']) > float(max_day_rate):
+                errs['dayRate'] = 'max_less_than_min'
```

Comment:
The name is confusing, but that's something we need to fix either way. I like reusing it here since it fits quite well and it helps keep the list of validation message keys smaller.